src/index.js

```diff
@@ -34,7 +34,7 @@ if (module.hot) {
       <AppContainer>
         <NextApp store={store} history={syncedHistory} />
       </AppContainer>
-      ), rootElement
+      ), rootElement,
```

Comment:
I'm curious about this. Did you use `` eslint --fix `` to fix those missing trailing comma?